app/controllers/work_controller.rb

```diff
@@ -257,6 +257,42 @@ def record_deed(work, deed_type, user)
     update_search_attempt_contributions
   end
 
+  def show
+    # Set meta information for work pages for better archival
+    @page_title = "#{@work.title} - #{@collection.title}"
+    @meta_description = "Historical work: #{@work.title} by #{@work.author} in the #{@collection.title} collection. #{@work.description}".truncate(160)
+    @meta_keywords = [@work.title, @work.author, @collection.title, "historical document", "digital archive"].compact.join(", ")
+    
+    # Generate structured data for work
+    @structured_data = {
+      "@context" => "https://schema.org",
+      "@type" => "Book",
```

Comment:
What other options are there for `@type` besides `Book`?  Most of our `work` records are letters, government forms, diaries, account books, or other records which are rarely described as "Books".